frontend/server/data/tinybird/pull-requests-data-source.ts

```diff
@@ -136,8 +136,8 @@ export async function fetchPullRequests(filter: ActivityCountFilter): Promise<Pu
   const previousMergedCount = previousMergedPRsSummary.data[0]?.activityCount || 0;
   const currentClosedCount = currentClosedPRsSummary.data[0]?.activityCount || 0;
   const previousClosedCount = previousClosedPRsSummary.data[0]?.activityCount || 0;
-  const currentTotalCount = currentOpenedCount + currentMergedCount + currentClosedCount;
-  const previousTotalCount = previousOpenedCount + previousMergedCount + previousClosedCount;
+  const currentTotalCount = currentOpenedCount - currentMergedCount + currentClosedCount;
```

Comment:
Didnt we agree just to have opened number here

Reply:
Yes, but like I said in my comment above, I had opened the PR before starting that discussion. I'm just waiting for the final confirmation about how it should be done and then I'll put the PR back in "ready to review" state.